voc/anomaly_detection.go

```diff
@@ -25,8 +27,10 @@
 
 package voc
 
+// AnomalyDetection analyzes the activity of a NetworkService (which includes DatabaseServices)
 type AnomalyDetection struct {
 	*Auditing
-	ApplicationLogging *ApplicationLogging `json:"applicationLogging"`
-	Enabled            bool                `json:"enabled"`
+	ApplicationLogging	*ApplicationLogging 	`json:"applicationLogging"`
```

Comment:
we should probably run `go fmt` on these files after generating them? Otherwise its hard to see the actual changes in the structs.

Reply:
I forgot that.  It would be cool if you could do this directly in the owl2java code, but I don't know how.